login-workflow/src/pages/create-account-invite/create-account-invite.component.html

```diff
@@ -51,6 +51,8 @@
         [showDefaultAccountDetails]="true"
         [(firstName)]="firstName"
         [(lastName)]="lastName"
+        [pageTitle]="accountDetails[0]?.pageTitle"
+        [pageInstructions]="accountDetails[0]?.pageInstructions"
```

Comment:
On the react side, we don't allow them to change the title on the first Account Details page, but I'm okay with this I think.